src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -260,6 +261,132 @@ public static void BuildChain_WithCertificatePolicy_NoMatch()
         }
 
         [Fact]
+        public static void BuildChain_MicrosoftDotCom_WithSystemRootCerts()
+        {
+            // Verifies that the microsoft.com certs build with only the certificates in the root store
+            // This is a canary test for BuildChain_MicrosoftDotCom_WithRootCertInUserAndSystemRootCertStores
+
+            using (var microsoftDotCom = new X509Certificate2(TestData.MicrosoftDotComSslCertBytes))
+            using (var chainHolder = new ChainHolder())
+            {
+                X509Chain chainValidator = chainHolder.Chain;
+                bool chainBuildResult = chainValidator.Build(microsoftDotCom);
+
+                StringBuilder builder = new StringBuilder();
+                foreach (var status in chainValidator.ChainStatus)
+                {
+                    builder.AppendFormat("{0} {1}{2}", status.Status, status.StatusInformation, Environment.NewLine);
+                }
+
+                Assert.True(chainBuildResult, "Chain build for microsoft.com cert failed. Reason: " + builder.ToString());
+            }
+        }
```

Comment:
I meant `chain.ChainPoicy.VerificationTime = new DateTime(2016, 7, 19, 19, 34, 0, DateTimeKind.Utc);` (or however many values you have to give to DateTime to make it happy).  (Revocation became Verification, Time became Date... you know... related... ish)